### ISSUE TITLE
[FEATURE] Suchfunnktion in den Akten

### ISSUE THREAD

**Issue by RA-crm** [jlawyerorg/j-lawyer-org#1036]
Die Suchfunktion bei "Akten Suchen" greift die Zeichenfolge auf und zeigt alle Akten an, die diese Zeichenfolge aufweisen. Nach dem es nun möglich ist, Akten einzelnen Gruppen zuzuweisen, wäre eine Suchfunktion die nach Gruppenzugehörigkeit filtert wünschenswert.

**Comment by j-dimension** [comment]
Wäre es hier evtl. schon hilfreich, die Gruppe in der Ergebnistabelle zu sehen?

**Comment by RA-crm** [comment]
Ja, denn mit der Ergebnistabelle, die man dann auch sortieren, wenn nicht sogar filtern können müsste, könnte dann die Akten einer Gruppe mit bestimmten Zeichenfolge gefunden werden.
